docs/README.md

```diff
@@ -1124,18 +1124,16 @@ export class HandleAvailability {
 
 #### Creating an event handler
 
-Event handlers can be easily created using the Booster CLI. There are two compulsory arguments that will need to be provided following the `boost new:event-handler` command, the first one will be the event handler name, and the other will be the name of the event that it will react to. For instance:
+Event handlers can be easily created using the Booster CLI command `boost new:event-handler`. There are two compulsory arguments: the event handler name, and the name of the event it will react to. For instance:
```

Comment:
What does `compulsory` mean here?

Reply:
Maybe... ??
```suggestion
Event handlers can be easily created using the Booster CLI command `boost new:event-handler`. There are two mandatory arguments: the event handler name, and the name of the event it will react to. For instance:
```

Reply:
`compulsory` means "not optional", but I don't like the word too. I like the Javi's suggestion